docs/user/keywords/index.rst

```diff
@@ -1783,6 +1773,47 @@ The two keywords SETENV and UPDATE_PATH can be used to manipulate the Unix envir
 
 	The whole thing is just a workaround because we can not use $PATH.
 
+.. _update_settings:
+.. topic:: UPDATE_SETTINGS
+
+The :code:`UPDATE_SETTINGS` keyword is a *super-keyword* which can be
+used to control parameters which apply to the Ensemble Smoother update
+algorithm. The :code:`UPDATE_SETTINGS`currently supports the two
+subkeywords:
+
+   OVERLAP_LIMIT
+        Scaling factor used when detecting outliers. Increasing
+      	this factor means that more observations will potentially be
+      	included in the assimilation. The default value is 3.00..
+
+	Including outliers in the Smoother algorithm can dramatically
+	increase the coupling between the ensemble members. It is
+	therefore important to filter out these outlier data prior to
+	data assimilation. An observation, \textstyle d^o_i, will be
+	classified as an outlier if
+
+	::
+
+		|d^o_i - \bar{d}_i| > \mathrm{ENKF\_ALPHA} \left(s_{d_i} + \sigma_{d^o_i}\right), 
```

Comment:
no need to change, but `\mathrm` can (and should) in most cases be simply `\text` or `\textnormal`.

`\mathrm` means that you want to go to roman font, which you should rarely (if ever) do.  `\text` simply switches to text mode.